docs/DeveloperGuide.md

```diff
@@ -63,7 +64,10 @@ Optionally, you can follow the [Using Checkstyle](UsingCheckstyle.md) document t
 ### Configuring the JavaScript coding style
 Our project follows the [Airbnb Javascript Style Guide](https://github.com/airbnb/javascript), the eslint configuration file is available at the root of the project. Please run a `npm run lint -- --fix frontend/src/**/*js` from the project root directory and fix all of the eslint errors before committing your code for final review.
 
-Eslint and its accompaning modules can be installed through NPM, so do ensure that you got it [installed](https://www.npmjs.com/get-npm) if you are working on the report.
+### Configuring the CSS coding style
+Our project follows the [OSS CSS Coding Standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Css.html), the stylelint configuration file is available at the root of the project. Please run a `npm run stylelint` from the project root directory and fix all of the stylelint errors before committing your code for final review.
+
+Eslint, Stylelint and their accompanying modules can be installed through NPM, so do ensure that you got it [installed](https://www.npmjs.com/get-npm) if you are working on the report.
```

Comment:
Preferably the javascript and css coding style can be combined together as they both use npm.
The command can also be combined to `npm run lint -- --fix frontend/src/**/*js && npm run stylelint` so that developers dont have to do it separately.